vrt/README.md

```diff
@@ -0,0 +1,114 @@
+# Visual testing
+
+We use `jest-image-snapshot` for a simple suite of visual regression tests (VRT). Note, that the current system is a work in progress and may change in the future.
+
+## Set up
+
+The suite will pick up and run any test file with the `*.vrt.js` extension. By default we turn every storybook scenario file into a single snapshot test. To add a new test, simply create a new `.scenario.js` file in the `__tests__` directory of a component.
+
+> Note, a temporary requirement is that the export `name` and file name of the scenario are identical.
+
+You can configure each scenario by adding a property matching the scenario name to the vrt config object in `vrt/config.js`.
```

Comment:
should we co-locate this with the tests? it seems a bit unintuitive to edit a configuration file to change a test case

Reply:
yeah i don't think the config object is a great solution. i went back and forth on a couple different approaches and ended up here.

i mentioned to chase in a previous comment how i'd like to move this type of configuration into the scenario files or into a new `.vrt.js` file for each component (similar to how i originally had it but only need to add if overriding default config or adding interactions).

the config is pretty simple for now though so i figured i could come back to that as a follow up effort in the interest of getting this out.